src/components/APIDoc/ApiDoc.tsx

```diff
@@ -38,16 +38,25 @@ export const ApiDoc: React.FunctionComponent<ApiDocProps> = props => {
                 <Text component={TextVariants.p}>
                     { openapi.info.description }
                 </Text>
-            </TextContent>
+            </TextContent>           
```

Comment:
Could you remove the space? I guess we need a linter to enforce these
```suggestion
            </TextContent>
```